inst/qml/TTestPairedSamples.qml

```diff
@@ -73,6 +73,10 @@ Form
 			name: "descriptivesPlots";						label: qsTr("Descriptives plots")
 			CIField { name: "descriptivesPlotsConfidenceInterval";	label: qsTr("Confidence interval") }
 		}
+		CheckBox
+        {
+        	name: "descriptivesPlotsRainCloud";		label: qsTr("Raincloud-like plots")
```

Comment:
Why do we say "Raincloud-like plots" again? Why not "Raincloud plots"? 😄

Reply:
Because you mentioned that we don't have plots that look like the original raincloud plots (our's don't really look like rainclouds anymore). But I can also change that, if you think it is confusing.

Reply:
Oh yeah. Maybe change it to just `Raincloud plots`, I guess it's close enough. We could theoretically think about an option that let's you choose between this configuration (group of scaters - group of boxplots - group of densities) and the "proper" raincloud (scatter - boxplot - density, scatter - boxplot - density), but not crucial for now I guess